lib/galaxy/datatypes/binary.py

```diff
@@ -1120,6 +1120,39 @@ def sniff( self, filename ):
 Binary.register_sniffable_binary_format("xlsx", "xlsx", Xlsx)
 
 
+class ExcelXls( Binary ):
+    """Class describing an Excel (xls) file"""
+    file_ext = "excel.xls"
+
+    def sniff( self, filename ):
+        mime_type = subprocess.check_output("file --mime-type '{}'".format(filename), shell=True).rstrip()
+        if mime_type.find("application/vnd.ms-excel") != -1:
+            return True
+        else:
+            return False
+
+    def get_mime( self ):
+        """Returns the mime type of the datatype"""
+        return 'application/vnd.ms-excel'
+
+    def set_peek( self, dataset, is_multi_byte=False ):
+        if not dataset.dataset.purged:
+            dataset.peek = "Microsoft Excel 2003 file"
```

Comment:
I can't tell if 2003 or 97 is better - maybe I'll just go with "Microsoft Excel XLS file" - https://msdn.microsoft.com/en-us/library/office/gg615597(v=office.14).aspx.